src/classes/AvoidHardcodingId - Copy.cls

```diff
@@ -0,0 +1,9 @@
+public with sharing class AvoidHardcodingId {
+
+	public void main(Account a) {
+		if(a.Id == '012500000009WAr') {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Hardcoding Ids is bound to break when changing environments.](https://app.codacy.com/manual/mani2246/apex-pmd-samples/pullRequest?prid=5681645)